Automattic/wp-calypso
# Add Akismet 2 year plans constants

## Proposed Changes

* Add constants for Akismet 2 year plans
* Add a few more groups of constants to make the checkout feature strings more robust (not needing to be edited as much in the future)
* Update upgrade paths to include the 2 year plans

## Testing Instructions

1. Checkout this branch or use the Calypso live link
2. Check out this diff in WPCOM: D125628-code
3. In your 0-sandbox.php file, add the following line
```
lang=php
define( 'USE_STORE_SANDBOX', true );
```
4. Purchase an Akismet 2 year plan (example checkout: https://wordpress.com/checkout/akismet/ak_plus_bi_yearly_1)
List of all 2 year plan slugs
* ak_plus_bi_yearly_1
* ak_plus_bi_yearly_2
* ak_plus_bi_yearly_3
* ak_plus_bi_yearly_4
* ak_ent_bi_yearly_1

While testing out checkout, make sure the feature slugs are correct for all bi-yearly and other Akismet products since the way feature slugs are selected was changed

![image](https://github.com/Automattic/wp-calypso/assets/65001528/077a9608-b8b8-4cf1-93d9-ec7ef0349f39)

5. Go to `/me/purchases` and select your Akismet 2 year plan
 
![image](https://github.com/Automattic/wp-calypso/assets/65001528/aa6f63ff-4573-47f6-88d7-d39bb436d9f2)

6. Make sure you can press the Upgrade and Renew buttons and you are taken to the correct checkout for both with no errors

Upgrade:
![image](https://github.com/Automattic/wp-calypso/assets/65001528/5c8d3b7e-4fdc-4dba-b230-31ca7ff89ef9)

Renew:
![image](https://github.com/Automattic/wp-calypso/assets/65001528/91bcbc18-5bf9-4b9f-92bc-ca2a443a5950)

## Pre-merge Checklist
